app/src/main/java/ch/heigvd/pro/b04/android/datamodel/Session.java

```diff
@@ -0,0 +1,49 @@
+package ch.heigvd.pro.b04.android.datamodel;
+
+import com.google.gson.annotations.SerializedName;
+
+public class Session {
```

Comment:
We do not need a Session object

Reply:
We need i tfor now, in order to get the Poll.